helm-vote/templates/test/ballot/test.yaml

```diff
@@ -0,0 +1,58 @@
+config:
+  target: "{{.Values.ballot_ip}}"
```

Comment:
Should be constructed using existing values rather than adding a new one

image: '{{- printf "%s/%s:%s" .Values.registry .Values.services.ballot.imageName .Values.tag -}}'